loris/img.py

```diff
@@ -215,21 +215,21 @@ def __getitem__(self, image_request):
 		return fp
 
 	@staticmethod
-	def _link(to,frum):
+	def _link(to, frum):
```

Comment:
I have the opposite problem (src, dest confuses me), but going with the descriptors used in the python docs would probably make sense: https://docs.python.org/2/library/os.html#os.symlink